stimela/cargo/cab/aimfast/src/run.py

```diff
@@ -26,15 +26,21 @@
     elif value is False:
         continue
 
-    if 'compare' in name:
+    if name in ['compare-images', 'compare-residuals', 'compare-models']:
         compare = []
         for i, val in enumerate(value):
             compare.append(val)
             # Compare models/images in pairs
             if i%2:
                 args += ['{0}{1} {2}'.format(cab['prefix'],
-                                            name, " ".join(compare))]
+                                             name, " ".join(compare))]
                 compare = []
+    elif name in ['compare-online']:
+        compare = []
+        for i, val in enumerate(value):
+            compare.append(val)
```

Comment:
These three lines seem a very confusing way of saying ``compare = list(enumerate(value))``.

Reply:
duly